
public class TestException {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		System.out.println("main started");
		Exception m1= new Exception();
		m1.exception1(4, null);
		m1.exception1(0, "Rahul");
		System.out.println("main ended");

	}

}
